plugins/Login/PasswordResetter.php

```diff
@@ -274,7 +280,7 @@ public function generatePasswordResetToken($user, $expiryTimestamp = null)
 
         $expiry = strftime('%Y%m%d%H', $expiryTimestamp);
         $token = $this->generateSecureHash(
-            $expiry . $user['login'] . $user['email'],
+            $expiry . $user['login'] . $user['email'] . $user['ts_password_modified'] . $resetTime,
```

Comment:
Is it possible to include something random here? like `Common::getRandomString()`?

Reply:
👍 I can use a random string instead of the reset time.